tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -732,14 +732,14 @@ public function getRenderListElementTests()
             ],
             [
                 '<td class="sonata-ba-list-field sonata-ba-list-field-array" objectId="12345">
-                    [1 => First] [2 => Second]
+                    [ 1 &nbsp;=>&nbsp; First , 2 &nbsp;=>&nbsp; Second ]
```

Comment:
Where does these space come from ?

I would expect 
```
[1&nbsp;=>&nbsp;First, 2&nbsp;=>&nbsp;Second]
```
There is no point adding `&nbsp;` if we're using a breakable space too.

BTW, the spaceless tag is deprecated, maybe changing this will help you.
https://twig.symfony.com/doc/2.x/deprecated.html

Reply:
Thank you for the deprecated notice! New extra spaces come from a lot of new lines and conditions. I can try to change such behaviour, but I think that it's ok. And I noticed that `&nbsp;` is useless in such a view.

![image](https://user-images.githubusercontent.com/617002/78678374-86aa9880-78f1-11ea-80ff-5cbeb9d4c774.png)

Reply:
So, `spaceless` removes spaces only between tags and not inside tags content

Reply:
You have the trim options https://symfony.com/blog/better-white-space-control-in-twig-templates#new-whitespace-trimming-options

Reply:
Missed this nice feature, thank you!